constants/images.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  login: require('assets/login.png'),
```

Comment:
Proszę dodaj lazy loading assetów

`LOGIN: () => require('assets/login.png')`

a potem używamy to jako `images.LOGIN()`